### PR TITLE
Small changes

### LINKED ISSUE #24: Make UI component for abolishing things clearer
I think it would be clearer if the abolish switches were either reversed (turn UC on/off) or if the color once activated is red instead of blue. Here's how it looks today:
image

image

### LINKED ISSUE #27: Use a different icon than check if someone skips About you section
e.g. a dash. Here's how it looks when going straight from policy to `Simulate on the population`:
image

### LINKED ISSUE #34: Add a footer
e.g.
>© 2021 The UBI Center. Let us know what you think! Email [policyengine@ubicenter.org](mailto:policyengine@ubicenter.org) or [file an issue on GitHub](http://github.com/ubicenter/uk-policy-engine/issues/new).

### LINKED ISSUE #35: Specify alpha or beta in the title

### LINKED ISSUE #38: Add intra-decile chart
Like this from the Lib Dem analysis:
image

### LINKED ISSUE #42: Add caveats about FRS limitations to population results

### LINKED ISSUE #43: Add caveats at the individual level
Potentially also link to other benefit calculators

### LINKED ISSUE #47: Fix non-incognito mode errors
For some reason, the site only works in incognito. I suspect this is because of how we're directing requests to the static site using a 404 handler; that this is causing the browser to think static site requests are all the index.html, caching it and using the index.html file in place of static resources improperly.

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Fixes #38
Fixes #42 
Fixes #47 
Fixes #24 (went with red rather than inverted)
Fixes #43 
Fixes #35 
Fixes #34 
Fixes #27

### CODE REVIEW

**Comment by nikhilwoodruff** [comment]
Fixes #24

File: client/src/js/pages/population-results/results.jsx [hunk 1]
```diff
@@ -34,10 +35,21 @@ function Chart(props) {
 	);
 }
 
+function PopulationResultsCaveats(props) {
+	return (
+		<Collapse defaultActiveKey={["1"]} ghost>
+			<Panel header={<><WarningFilled style={{color:"orange"}} />  Warning</>} key="1">
+				<p>Figures calculated here are based on the Family Resources Survey, uprated from 2018-19, using the OpenFisca-UK tax-benefit microsimulation model. Results may differ from actual results due to under-reporting of incomes and benefit receipts, and policy not covered by the model. See the <a href="https://github.com/PSLmodels/openfisca-uk/issues">repository</a> for more information.</p>
```
Comment (MaxGhenis) [inline client/src/js/pages/population-results/results.jsx]: Uprated to 2021 right

Also link to repo home vs issues

File: server/populations/charts.py [hunk 2]
```diff
@@ -183,3 +184,107 @@ def age_chart(baseline, reformed):
...
+                "decile": "All",
+                "Outcome": name,
+            }
+        )
+        l.append(all_row)
+    return pd.concat(l).reset_index()
+
+
+INTRA_DECILE_COLORS = (
+    "#616161",
```
Comment (MaxGhenis) [inline server/populations/charts.py]: Could you make these colors descriptive constants too

File: client/src/js/pages/population-results/results.jsx (hunk 1 above)
Comment (nikhilwoodruff) [reply client/src/js/pages/population-results/results.jsx]: Oh yep that's right

File: server/populations/charts.py (hunk 2 above)
Comment (nikhilwoodruff) [reply server/populations/charts.py]: Done

File: client/src/js/pages/population-results/results.jsx (hunk 1 above)
Comment (nikhilwoodruff) [reply client/src/js/pages/population-results/results.jsx]: Done

**Comment by MaxGhenis** [comment]
please make this one red too
image

**Comment by nikhilwoodruff** [comment]
Done, merging to update progress

**Pull request merged**